src/Rings/AbelianClosure.jl

```diff
@@ -459,6 +469,7 @@ function minimize(::typeof(CyclotomicField), a::nf_elem)
   return minimize(CyclotomicField, [a])[1]
 end
 
+#TODO: document this!
```

Comment:
this function is ambigous: the element defines an order - which has a conductor as an order, and a field. The notions are different. So a comment/ doc would be important